packages/flutter_tools/lib/src/runner/flutter_command_runner.dart

```diff
@@ -71,6 +71,10 @@ class FlutterCommandRunner extends CommandRunner<Null> {
         negatable: true,
         hide: !verboseHelp,
         help: 'Whether to use terminal colors.');
+    argParser.addFlag('skip-version-check',
```

Comment:
Maybe call it `version-check`, default it to true, and make it negatable?  So your use would look like `--no-version-check`

Reply:
Good idea; didn't realise negate allowed a `no-` prefix :)